Xamarin.Essentials/Calendar/Calendar.ios.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using EventKit;
+using Foundation;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Calendar
+    {
+        static async Task<IEnumerable<DeviceCalendar>> PlatformGetCalendarsAsync()
+        {
+            await Permissions.RequireAsync(PermissionType.CalendarRead);
+
+            EKCalendar[] calendars;
+            try
+            {
+                calendars = CalendarRequest.Instance.Calendars;
+            }
+            catch (NullReferenceException ex)
+            {
+                throw new Exception($"iOS: Unexpected null reference exception {ex.Message}");
+            }
+            var calendarList = new List<DeviceCalendar>();
```

Comment:
Change to:
var calendarList = (from calendar in calendars
                              select new DeviceCalendar
                {
                    Id = t.CalendarIdentifier,
                    Name = t.Title
                }).ToList();

Reply:
Addressed in: https://github.com/builttoroam/Essentials/pull/43